discovery-infra/test_infra/tools/static_network.py

```diff
@@ -15,6 +15,26 @@ def generate_macs(count):
             for _ in range(count)]
 
 
+def generate_day2_static_network_data_from_tf(tf_folder, num_workers):
```

Comment:
`num_workers` are additional workers, aren't they? Meaning not all workers, but only day 2 ones. I think it's worth reflecting in the name.

Reply:
fixed